cypress/support/constants/constants.js

```diff
@@ -457,6 +458,10 @@ module.exports = {
   SECONDARY_THIRD_PARTY_APP_ID: 'secondary3rdPartyAppId',
   SECONDARY_APPID_MISSING_ERROR:
     '`Unable to find the ${envAppIdKey} value in the env, please add the value in configModule/constants/config.json`',
+  TRIGGER_EVENT_MISSING:
+    'triggerEvent request override function to set event values not implemented by underlying platform',
+  TRIGGER_EVENT_REQUEST: 'Request sent to platform to trigger event: ',
+  TRIGGER_EVENT_SUCCESS: 'Event triggered successfully',
 };
 function getSanityReportPath() {
```

Comment:
Log line for 461 does not make sense 
Line 463 - Log doesnt make sense . In this case who is triggering the event ? Has there been a case where events are sent to the platform  ?